src/Shared/HttpSys/RequestProcessing/NativeRequestContext.cs

```diff
@@ -391,8 +506,8 @@ private void GetUnknownHeadersHelper(IDictionary<string, StringValues> unknownHe
                 }
                 pUnknownHeader++;
             }
+            }
```

Comment:
? Formatting

Reply:
That looks weird, I did not change this line and it looks ok in my editor. Double checking.